tests/manage/storageclass/test_create_multiple_sc_with_same_pool_name.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+import logging
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import defaults
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import (
+    create_ceph_block_pool,
+    create_rbd_secret, create_cephfs_secret,
+    create_project
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.usefixtures(
+    create_project.__name__,
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+)
+class TestCreateMultipleScWithSamePoolName(ManageTest):
+    """
+    Create Multiple Storage Class with same pool name
+    """
+    @pytest.mark.parametrize(
+        argnames="interface",
+        argvalues=[
+            pytest.param(
+                *["RBD"], marks=pytest.mark.polarion_id("OCS-622")
+            ),
+            pytest.param(
+                *["CEPHFS"], marks=pytest.mark.polarion_id("OCS-623")
+            )
+        ]
+    )
+    def test_create_multiple_sc_with_same_pool_name(self, interface):
+        """
+        This test function does below,
+        *. Creates multiple Storage Classes with same pool name
+        *. Creates PVCs using each Storage Class
+        *. Mount each PVC to an app pod
+        *. Run IO on each app pod
+        """
+        # Create 5 Storage Classes with same pool name
+        sc_obj_list = []
+        sc_list = []
+        for i in range(1, 3):
+            log.info(f"Creating a {interface} storage class")
+            if interface == "RBD":
+                interface_type = constants.CEPHBLOCKPOOL
+                secret = self.rbd_secret_obj.name
+                interface_name = self.cbp_obj.name
+            else:
+                interface_type = constants.CEPHFILESYSTEM
+                secret = self.cephfs_secret_obj.name
+                interface_name = helpers.get_cephfs_data_pool_name()
+            sc_obj = helpers.create_storage_class(
+                interface_type=interface_type,
+                interface_name=interface_name,
+                secret_name=secret,
+            )
+            log.info(
+                f"{interface}StorageClass: {sc_obj.name} created successfully"
+            )
+            sc_obj_list.append(sc_obj)
+            sc_list.append(sc_obj.name)
+
+        # Create PVCs using each SC
+        pvc_obj_list = []
+        pvc_list = []
+        for i in range(1, 3):
+            log.info(f"Creating a PVC using {sc_list[i-1]}")
+            pvc_obj = helpers.create_pvc(sc_list[i - 1])
+            log.info(
+                f"PVC: {pvc_obj.name} created successfully using"
+                f" {sc_list[i-1]}"
+            )
+            pvc_list.append(pvc_obj.name)
+            pvc_obj_list.append(pvc_obj)
+
+        # Create app pod and mount each PVC
+        pod_obj_list = []
+        pod_list = []
+        for i in range(1, 3):
+            if interface == "RBD":
+                interface_type = constants.CEPHBLOCKPOOL
```

Comment:
```
                interface_type = constants.CEPHBLOCKPOOL
```
This can be avoided by following [comment](https://github.com/red-hat-storage/ocs-ci/pull/516/files#r311517741)

Reply:
Done

Reply:
Done